secret-handshake.md

```diff
@@ -14,13 +14,14 @@ binary decide to come up with a secret "handshake".
 10000 = Reverse the order of the operations in the secret handshake.
 ```
 
-```
-handshake = SecretHandshake.new 9
-handshake.commands # => ["wink","jump"]
+Here's a couple of examples:
+
+Given the input 9, the function would return the array
+["wink","jump"]
```

Comment:
it's minor, I know, but there is no space after this comma but is a space after the other. Consistency would be good.